ibis/expr/api.py

```diff
@@ -3844,37 +3844,93 @@ def _time_sub(left, right):
 }
 
 
-def join(left, right, predicates=(), how='inner'):
-    """Perform a relational join between two tables. Does not resolve resulting
-    table schema.
+def _dedup_join_columns(expr, *, left, right, rename_left, rename_right):
```

Comment:
should be be located elsewhere and imported here?

Reply:
Sounds good, I'll move to `operations/relations.py`.

Reply:
Done.